lib/meadow_web/schema/types/data/controlled_vocabulary_types.ex

```diff
@@ -15,50 +16,46 @@ defmodule MeadowWeb.Schema.Data.ControlledTermTypes do
       resolve(&ControlledVocabulary.code_list/3)
     end
 
-    @desc "NOT YET IMPLEMENTED Get the label for a controlled_term by its id"
+    @desc "Get the label for a controlled_term by its id"
     field :fetch_controlled_term_label, :controlled_value do
       arg(:id, non_null(:id))
       middleware(Middleware.Authenticate)
 
-      resolve(fn %{id: _id}, _ ->
-        {:ok, nil}
-      end)
+      resolve(&AuthoritiesSearch.fetch_label/2)
     end
 
     @desc "Get the label for a coded_term by its id and scheme"
     field :fetch_coded_term_label, :coded_term do
       arg(:id, non_null(:id))
       arg(:scheme, non_null(:code_list_scheme))
       middleware(Middleware.Authenticate)
-      resolve(&ControlledVocabulary.fetch_controlled_term_label/3)
+      resolve(&ControlledVocabulary.fetch_coded_term_label/3)
     end
 
-    @desc "NOT YET IMPLEMENTED Get the label for a code list item by its id"
+    @desc "Get a list of authority search results by its authority"
     field :authorities_search, list_of(:controlled_value) do
       arg(:authority, non_null(:coded_term_input))
```

Comment:
Does the `authority` param here really have to be a `:coded_term_input` or can it just be a `:string` or an `:id`? The `scheme` part is never going to be anything but `AUTHORITY`, so why not just let people call
```
query{
  authoritiesSearch(authority:"lcnaf", query:"kricfalusi") {
    id
    label
  }
}
```

instead of 
```
query{
  authoritiesSearch(authority:{id:"lcnaf", scheme:AUTHORITY}, query:"kricfalusi") {
    id
    label
  }
}
```

Reply:
Done!